[README.md] Enhance formatting and wording

fvsamson opened the pull request:
- Capitalise names
- Try to be more concise WRT copyright (notices)
- Full stop at the end of all full sentences (i.e. comprising a subject, object and verb).
- Etc.

fvsamson reviewed (commented):
@xoryouyou, please review, especially if the content of the lines I expanded is still correct.

fvsamson commented:
BTW, while the TR list is looking good, for IT-GS some obvious and some subtle flaws are displayed:
- All titles have `.pdf.clean.pdf` appended.
- Ordering is screwed up at a few places, because "10" is sorted before "1\s".  Likely a special rule for sorting strings containing two digit numbers after their single digit counterparts is required to fix that.
- I wonder if there are some documents missing, because there are gaps in the number ranges:
  * APP.1.3
  * APP.3.5
  * APP.4.1
  * APP.4.5
  * APP.5.1
  * CCON.4
  * CCON.5
  * IND.2.5
  * IND.2.6
  * IND.3.1
  * INF.3
  * INF.4
  * OPS.1.2.1
  * OPS.2.1
  * OPS.3.1
  * SYS.1.4
  * SYS.4.2
   
   I have not checked if these really do not exist (any longer?).

P.S.: Please feel free to alter anything you deem unfit, i.e. do not take the detour to request changes from me when something clearly does not make sense or (technically) does not work.  But please do ask if my consideration resulting in a change is not obvious for you.

xoryouyou commented:
> BTW, while the TR list is looking good, for IT-GS some obvious and some subtle flaws are displayed:
> 
> * All titles have `.pdf.clean.pdf` appended.

Oh yeah this is a leftover from a version of the repo when I still used the bundles [this](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Grundschutz/IT-GS-Kompendium_Einzel_PDFs_2023/Zip_Datei_Edition_2023.html) .zip file for all PDFs from the BSI.
see
![image](https://github.com/user-attachments/assets/1b291c61-320a-462c-ac34-fb6de1103da7)

I'll fix it directly on main 😄 

> * Ordering is screwed up at a few places, because "10" is sorted before "1\s".  Likely a special rule for sorting strings containing two digit numbers after their single digit counterparts is required to fix that.
> * I wonder if there are some documents missing, because there are gaps in the number ranges:
>   
>   * APP.1.3
>   * APP.3.5
>   * APP.4.1
>   * APP.4.5
>   * APP.5.1
>   * CCON.4
>   * CCON.5
>   * IND.2.5
>   * IND.2.6
>   * IND.3.1
>   * INF.3
>   * INF.4
>   * OPS.1.2.1
>   * OPS.2.1
>   * OPS.3.1
>   * SYS.1.4
>   * SYS.4.2
>   
>   I have not checked if these really do not exist (any longer?).

Yeah it really looks like that on the Grundschutz Bausteine Page

https://www.bsi.bund.de/DE/Themen/Unternehmen-und-Organisationen/Standards-und-Zertifizierung/IT-Grundschutz/IT-Grundschutz-Kompendium/IT-Grundschutz-Bausteine/Bausteine_Download_Edition_node.html
![image](https://github.com/user-attachments/assets/5b56d925-da1e-4fa4-a75f-337bccfa419c)

But thanks for pointing it out. I blindly trust the output of the scraper and have only checked random samples and not all links. Always good to have a second pair of eyes.


> P.S.: Please feel free to alter anything you deem unfit, i.e. do not take the detour to request changes from me when something clearly does not make sense or (technically) does not work. But please do ask if my consideration resulting in a change is not obvious for you.

xoryouyou commented:
I broke the pull request by fixing the suffix `.pdf.clean.pdf` issue on main.
Had to merge it into here manually now everything looks fine again 👍

fvsamson commented:
> Had to merge it into here manually now everything looks fine again 👍

Thanks!

A bit nitpicking remarks (HTH, and is not perceived as lecturing):
- "Manually"
  GitHub's web-frontend automatically offers to update a feature branch with the recent commits of the target branch in a PR, if merge conflicts arise.  For trivial merge conflicts, it even offers a slightly awkward and clumsy text editor to resolve these conflicts manually without leaving the web-frontend.  If you want that "one click" offer to update the feature branch with the recent commits of the target branch to be always present (i.e. not only when merge conflicts have been introduced by new commits to the target branch), there is a switch for that in the repo settings, which I usually switch on.
  BTW, another convenient feature is "automatically delete a feature branch when it was merged", which I always switch on; GitHub offers a button to restore an automatically deleted feature branch for weeks on the web-page of the corresponding PR.  There are some other settings I usually configure, e.g. setting up a simple, "classic" branch protection rule.  The nice thing is, that most of these settings configure Git features, hence these settings also effective when pushing to a repo at GitHub by `git` (the command line tool), using a local Git-GUI-frontend or a local GitHub-GUI-Client which utilises GitHub's API.
  If you want me to adapt these settings as I usually do (you can still alter anything later), I did that so often with non-work repos at GitHub that it takes me about 10 minutes, if you provide me with "admin" access to this repo (less does not allow for configuring repo settings, unfortunately).  IIRC one can limit the visibility of co-admins for personal repos (I have more experience with the access settings for repos in a GitHub "organisation", but IMO this is not currently needed here), or you revert my access after configuring the settings.
- I started utilising repos at GitHub as the real upstream- / tip-repo (instead of my local one) for non-work projects years ago and ended up using GitHub's web-frontend for all simple actions.  Only non-trivial merge conflicts (i.e. when the conflict editor is not offered in GitHub's web-frontend) and script based changes (usually affecting multiple files) are reasons for me to clone locally, alter and then push the changes right back to the "tip" repo at GitHub.  Thus, if my local mass storage dies, nothing is lost, in contrast to all other data for which all changes since the last backup are lost.
- Hence I am glad you resolved this merge conflict and would (have) actively ask(ed) you to resolve it, if it is non-trivial for GitHub (i.e. the conflict editor is not offered).  OTOH, for trivial merge conflicts we can sure handle that as usual: It is the feature branch's maintainers business to resolve merge conflicts, if changes in the target branch create some.  But "yes", I would personally like to try avoiding to locally clone this repo as long this is conveniently feasible.

-----------------------------------------

For some reason using GitHub's review functionality (which is really nice) is sometimes not visible for others; suspects are "contributor must be given a(ny?) role for this repo", or (less likely) "both accounts must be in the same GH-'org'" which implies the prior point.

Apparently this is the case here, hence I tried to visually reconstruct my review in look&feel by copy&pasting it and adapting the resulting Markdown (see [this screenshot further up on this page](https://github.com/user-attachments/assets/a3587b6f-fe31-4ebf-b2a8-7e3d4c64f40d), as I see it):
<br />

@fvsamson started a review (2025-05-25)&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;**`Pending`**

[**`README.md`**, lines 42 - 45](https://github.com/xoryouyou/bsi-technical-guidelines-markdown/pull/6/files/1013b22879af96417c0872ad86fdd24becb9dc62#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R42-R45):
```
#### Details
* `scrape_pdf_links.py` was used to loop over all TR pages from `data/tr-list.txt` and extract links for the PDFs on them and store them in `data/pdf_links.txt`.
* `download_pdfs.py` did download them into the `/pdf` folder.
* `convert.sh` was used to loop over all PDFs and convert them using Marker and Ollama.
```
> @fvsamson&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;**`Pending`**
> Why do these three bullet points utilise past tense?
>
